plugins/catalog-backend-module-azure/src/providers/AzureDevOpsEntityProvider.test.ts

```diff
@@ -105,9 +105,13 @@ describe('AzureDevOpsEntityProvider', () => {
     await (taskDef.fn as () => Promise<void>)();
 
     const expectedEntities = codeSearchResults.map(item => {
-      const url = encodeURI(
-        `${expectedBaseUrl}/_git/${item.repository.name}?path=${item.path}`,
-      );
+      const url = item.branch
+        ? encodeURI(
+            `${expectedBaseUrl}/_git/${item.repository.name}?path=${item.path}&version=GBmybranch`,
```

Comment:
This can be item.branch right?

```suggestion
            `${expectedBaseUrl}/_git/${item.repository.name}?path=${item.path}&version=GB${item.branch}`,
```